cordova-app/package.json

```diff
@@ -5,30 +5,22 @@
   "homepage": "./",
   "cordova": {
     "platforms": [
-      "browser",
       "ios",
       "android"
     ],
     "plugins": {
       "cordova-plugin-whitelist": {},
-      "cordova-plugin-geolocation": {
-        "GEOLOCATION_USAGE_DESCRIPTION": "Geovation needs to know where you are so it can place your photo in the map"
-      },
-      "cordova-plugin-splashscreen": {},
-      "cordova-plugin-statusbar": {},
+      "cordova-plugin-camera-with-exif": {},
```

Comment:
are you sure that it doesn't need "ANDROID_SUPPORT_V4_VERSION" ?

Reply:
I used Cordova to remove and add plugins.
I also tried to run app on android devices and it is working.